logagg/formatters.py

```diff
@@ -205,6 +206,7 @@ def basescript(line):
               u'level': u'warning',
               u'timestamp': u'2018-02-07T06:37:00.297610Z',
               u'type': u'log'},
+     'event': u'exited via keyboard interrupt',
```

Comment:
the event value should look like a variable (even in examples to set the right expectation to the reader)